conans/client/client_cache.py

```diff
@@ -12,7 +12,8 @@
 from conans.model.profile import Profile
 from conans.model.ref import ConanFileReference
 from conans.model.settings import Settings
-from conans.paths import SimplePaths, CONANINFO, PUT_HEADERS
+from conans.paths import SimplePaths, CONANINFO, PUT_HEADERS, _check_ref_case,\
```

Comment:
maybe _check_ref_case as an static method of the SimplePaths to avoid the private import?